<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Point Cloud Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="PointCloudRenderer" name="rendererWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QDockWidget" name="dbDockWidget">
   <property name="windowTitle">
    <string>Viewport Objects</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QTreeWidget" name="dbTreeWidget">
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>0</height>
     </size>
    </property>
    <column>
     <property name="text">
      <string>Viewport Objects</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionResetView"/>
   </widget>
   <widget class="QMenu" name="menuViewport">
    <property name="title">
     <string>Viewport</string>
    </property>
    <addaction name="actionSave_Viewport_As_Object"/>
    <addaction name="actionSave_Viewport_with_User_defined_co_ords"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuViewport"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Alt+F4</string>
   </property>
  </action>
  <action name="actionResetView">
   <property name="text">
    <string>Reset View</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionSave_Viewport_As_Object">
   <property name="text">
    <string>Save Viewport As Object</string>
   </property>
  </action>
  <action name="actionSave_Viewport_with_User_defined_co_ords">
   <property name="text">
    <string>Save Viewport with User-defined co-ords</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PointCloudRenderer</class>
   <extends>QWidget</extends>
   <header>pointcloudrenderer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
